src/reply-password/reply-password.c

```diff
@@ -95,7 +95,5 @@ int main(int argc, char *argv[]) {
         r = send_on_socket(fd, argv[2], packet, length);
```

Comment:
BTW, if `argv[1]` is "1", then `length` includes the last NUL. However, if `argv[1]` is "0", then `length` _does not_ include NUL. Is it intended?

Reply:
yeah, in the "+" case we send a payload: the password we just read. In the "-" case we send no payload, but the password reception was cancelled.

The "+" payload is theroetically a list of individually NUL terminated passwords, one after the other. Hence it matters to include the NUL byte in the datagram, so that we can distuingish the case "one empty password sent" with "no password sent", as the former would be encoded in a single NUL byte, while the latter  as no bytes.

Reply:
I see. Thank you for the explanation.